website/blog/2020-02-31-version-60.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Announcing React Native 0.60
+author: Ryan Turner
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://twitter.com/turnrye
+authorImageURL: https://avatars0.githubusercontent.com/u/701035?s=460&v=4
+authorTwitter: turnrye
+category: announcements
+---
+
+After months of hard work from hundreds of contributors, the React Native Core team is proud to announce the release of version 0.60. This release handles significant migrations for both Android and iOS platforms, and many issues are resolved too. This blog post covers the highlights of the release. As always though, refer to the changelog for more detailed information. Finally, thank you contributors for helping us to make this milestone!
+
+## Focus on Accessibility
+
+There have been many improvements to the accessibility APIs, like `announceForAccessibility`, plus improvements to roles, action support, flags, and more. Accessibility is a complex science, but we hope these improvements make it a bit easier to be an A11Y. Be sure to check [React Native Open Source Update June 2019](https://facebook.github.io/react-native/blog/2019/06/12/react-native-open-source-update#meaningful-community-contributions) for more details of these changes.
```

Comment:
Could you list the ones that were merged for 0.60 here in the blog post directly?